proto/greptime/v1/meta/heartbeat.proto

```diff
@@ -60,6 +66,8 @@ message RegionStat {
   int64 approximate_rows = 5;
   // engine name
   string engine = 6;
+  // Indicate region's state.
+  RegionState state = 7;
```

Comment:
Question：
1. Do we need the Unknown role? Is it enough to use `None` to represent it?
2. I think `Readonly/Writeable` is a role rather than a state, for example in the future, we will also have a datanode that is specifically responsible for compaction, at which time we can add a `Compactor` role.